packages/tracing/src/browser/request.ts

```diff
@@ -186,9 +187,12 @@ export function fetchCallback(
     return;
   }
 
-  const activeTransaction = getActiveTransaction();
-  if (activeTransaction) {
-    const span = activeTransaction.startChild({
+  const currentScope = getCurrentHub().getScope();
+  const currentSpan = currentScope ? currentScope.getSpan() : undefined;
+  const activeTransaction = currentSpan ? currentSpan.transaction : undefined;
```

Comment:
```suggestion
  const currentScope = getCurrentHub().getScope();
  const currentSpan = currentScope && currentScope.getSpan();
  const activeTransaction = currentSpan && currentSpan.transaction;
```

Ditto with the logic below.